index.html

```diff
@@ -33,6 +33,274 @@
+          </p>
+          <p>
+            Information that is not currently rendered to the screen SHOULD be obscured in captures
+            unless the application has been specifically authorized to access that content (e.g.
+            through means such as
+            <a data-cite="screen-capture#elevated-permissions">elevated permissions</a>).
+          </p>
+        </dd>
+      </dl>
+    </section>
+
+    <section>
+      <h2>ScreenCaptureMediaStreamTrack</h2>
+      {{MediaDevices/ScreenCaptureMediaStreamTrack}} extends {{MediaStreamTrack}} and
+      corresponds to a <a data-cite="screen-capture#dfn-monitor">monitor</a>.
+      It additionally provides a connection to the window placement API [[!window-placement]]
+      with the {{MediaDevices/ScreenCaptureMediaStreamTrack/screenDetailed}} function with which metadata about the
+      captured <a data-cite="screen-capture#dfn-monitor">monitor</a> can be retrieved.
+      <pre class="idl">
+        [Exposed=Window]
+        interface ScreenCaptureMediaStreamTrack : MediaStreamTrack {
+          ScreenDetailed screenDetailed();
+        };
+      </pre>
+
+      <dl data-link-for="ScreenCaptureMediaStreamTrack" data-dfn-for="ScreenCaptureMediaStreamTrack" class="methods">
+        <dt>
+          <dfn>screenDetailed</dfn>
+        </dt>
+        The {{screenDetailed()}} method returns a {{ScreenDetailed}} object [[!window-placement]]
+        that corresponds to the <a data-cite="screen-capture#dfn-monitor">monitor</a> that
+        is captured by the {{ScreenCaptureMediaStreamTrack}} object.
+      </dl>
+    </section>
+
+    <section>
+      <h2>Misc</h2>
+      This specification extends the sections
+      <a data-cite="screen-capture#hidden-display-surfaces">Closed and Minimized Display Surfaces</a>, and
+      <a data-cite="screen-capture#deviceId">Device Identifiers</a>
+      of {{MediaDevices/getDisplayMedia()}} to include {{MediaDevices/getAllScreenMedia()}}.
+    </section>
+  </section>
+  <section id="privacyindicatorsrequirements">
+    <h2>
+      Privacy Indicator Requirements
+    </h2>
+    <p>
+      References in this specification to [[\devicesLiveMap]], and
+      [[\devicesAccessibleMap]] refer to the
+      definitions already created to support Privacy Indicator Requirements for
+      {{MediaDevices/getDisplayMedia()}}.
+    </p>
+    <p>
+      This specification extends the <a data-cite="screen-capture#privacy-indicator-requirements">
+      Privacy Indicator Requirements</a> of
+      {{MediaDevices/getDisplayMedia()}} to include {{MediaDevices/getAllScreenMedia()}}.
+      In addition to these requirements, <a href="#dfn-user-agent">user agents</a> MUST
+      ensure that privacy indicators are visible at all times and that dismissal of the indicators
+      must not be persisted. The privacy indicators must inform about the <a data-cite="url#concept-url-origin">origin</a>
```

Comment:
Is this copied from somewhere? Do you perhaps want to mention that only the user may dismiss the indicators?

Reply:
"Is this copied from somewhere?": no, i added this in response to your comment that there is not much explicit information about the notification. why?

"Do you perhaps want to mention that only the user may dismiss the indicators?": makes sense. added.